docs/modules/mongodb_watchdog.md

```diff
@@ -1 +1,15 @@
 # Logger: `mongodb_watchdog`
+
+The module uses a separate database, using the logger database alias in settings. Do NOT point that alias to the same database as default, because the module drops the logger database when uninstalling, which would drop all your other data with it.
```

Comment:
Wrap machine names like `logger` or `default` in backticks to make it clear these are file names / code symbols.